style/swift/README.md

```diff
@@ -14,3 +14,5 @@ Swift
 * Group methods into specific extensions for each level of access control
 * When capitalizing acronyms or initialisms, follow the capitalization of the
   first letter.
+* Break up long initializers or function calls by adding linebreaks between
```

Comment:
We typically start our guidelines with "Use", "Prefer", "Avoid", etc to better communicate how strict the guideline is. I have noticed that the Swift ones don't follow this pattern and I believe it'd be better if they did. For instance, it'd be:
"Prefer breaking up.." instead of "Break up...".